README.md

```diff
@@ -25,6 +25,10 @@ Sample message
       "id":"test",
       "name":"test"
    },
+   "stream":{ 
+      "id":"streamid",
```

Comment:
maybe n UUID since this should be a example

Reply:
yes